docs/Removing dividers support from layout components.md

```diff
@@ -0,0 +1,95 @@
+# Removing `dividers` support from layout components
+
+Following the update to SEEK&rsquo;s Browser Support Policy, we are now able to leverage [CSS gap] in Braid, and more importantly, our layout components. This will be landing in Braid in the v33 release.
+
+Previously, to enable gap-like behaviour layout components iterated over their children, wrapping them in container elements and applying padding. The trade off of this technique was an increased number of DOM elements and the introduction of unwanted space if the child element was hidden or returned `null` — requiring developers to hoist logic to the parent component.
+
+Migrating to CSS gap mitigates all of the above trade offs.
+
+However, because `Stack` and `Tiles` no longer iterate over their children, the `dividers` feature is no longer feasible to implement centrally.
+
+While we could have conditionally maintained this behaviour, it would have resulted in inconsistent edge cases when using `dividers` and not, e.g. if a child component rendered nothing or a hidden element, the `divider` would still be rendered, resulting in an inconsistent layout.
```

Comment:
Is `and not` meant to be here? Could also be split into a separate sentence.
```suggestion
While we could have conditionally maintained this behaviour, it would have resulted in inconsistent edge cases when using `dividers`. E.g. if a child component rendered nothing or a hidden element, the `divider` would still be rendered, resulting in an inconsistent layout.
```